include/common/attributes.rs

```diff
@@ -1,14 +1,16 @@
+use std::ffi::{c_int, c_uint, c_ulonglong};
+
 #[inline]
-pub unsafe extern "C" fn ctz(mask: libc::c_uint) -> libc::c_int {
-    return mask.trailing_zeros() as i32;
+pub fn ctz(mask: c_uint) -> c_int {
```

Comment:
For these functions that specify their types with the C numeric types, are we able to convert them to the Rust numeric types at this phase? If we're cleaning up these functions, may as well use more idiomatic Rust types if they work for all of our targets.

Reply:
I wasn't sure yet either.  In some PRs I think I may have changed `uint8_t` to `u8` in some places and things like that, but so far I've left the `ffi::c_*` types alone.